package.json

```diff
@@ -35,14 +35,19 @@
     "grunt-release": "^0.7.0",
     "jquery": "^2.1.1",
     "jscs": "^2.3.3",
-    "jsdom": "^6.5.1",
     "qunit": "^0.7.5",
     "raphael": "^2.1.0",
-    "xmldom": "^0.1.19"
+    "slimerjs": "^0.9.6"
   },
   "scripts": {
     "start": "grunt stage",
-    "test": "grunt test"
+    "lint": "grunt jshint",
+    "qunit": "grunt test",
+    "generate:current": "slimerjs ./tools/generate_svg_images.js ../build ./build/images/current",
+    "generate:blessed": "slimerjs ./tools/generate_svg_images.js ../releases ./build/images/blessed",
+    "generate": "npm run generate:current && npm run generate:blessed",
+    "diff": "./tools/visual_regression.sh",
+    "test": "npm run lint && npm run qunit && npm run generate && npm run diff"
```

Comment:
Here, I think it should just be `npm run generate:current`, no? Otherwise it won't catch the regression errors.